src/rest-server/src/models/job.js

```diff
@@ -434,6 +434,10 @@ class Job {
   }
 
   generateDockerContainerScript(data, idx) {
+    let paiMachineList = '';
+    if (fs.existsSync('/pai-cluster-config/cluster-configuration.yaml')) {
```

Comment:
In fact, this is a pretty slow operation, at least we could move it into an azRDMA condition block.

Reply:
Yes, agree with you that the solution is solw. But the cluster-configuration file is not a static file. And it will be updated, if admin perform the config update command. Are there any solution to update the config for a specify time period.